client-js/app/elements/selectable-table.html

```diff
@@ -15,7 +15,7 @@
       is: 'selectable-table',
       extends: 'tbody',
       behaviors: [
-        Polymer.IronSelectableBehavior
+        Polymer.IronMultiSelectableBehavior
```

Comment:
Probably should leave this multi-selection for later, since we currently only support single selection.